tests/loop5.py

```diff
@@ -0,0 +1,18 @@
+def with_break_for():
+    i: i32
+    for i in range(4):
+        print(i)
+        break
+    else:
+        print(10)
+
+def with_break_while():
+    i: i32 = 0
+    while i < 4:
+        print(i)
+        break
+    else:
+        print(10)
```

Comment:
Along with the print statement, why don't we test the outputs using assert statements?
Something like: loop_01.py, loop_02.py, ...